dp/cloud/python/magma/configuration_controller/tests/unit/test_response_processor.py

```diff
@@ -348,21 +348,31 @@ def test_channel_params_set_on_grant_response(self):
         self.assertEqual(max_eirp, grant.max_eirp)
 
     @parameterized.expand([
-        (REGISTRATION_REQ, registration_requests),
-        (SPECTRUM_INQ_REQ, spectrum_inquiry_requests),
-        (GRANT_REQ, grant_requests),
-        (HEARTBEAT_REQ, heartbeat_requests),
-        (RELINQUISHMENT_REQ, relinquishment_requests),
-        (DEREGISTRATION_REQ, deregistration_requests),
+        (REGISTRATION_REQ, registration_requests, ResponseCodes.DEREGISTER.value, None, CbsdStates.UNREGISTERED.value),
```

Comment:
I'm wondering if we could somehow merge first to parameters, since the first one seems to always imply the second one, that way we would have 4 parameters instead of 5 and be more readable

Reply:
Created a simple map that links request types with fixtures.
All callers retrieve the fixture from the map based on request type key.